packages/xs-vat-worker/xs-lin.mk

```diff
@@ -0,0 +1,41 @@
+MODDABLE=$(PWD)/moddable
+TOOLS=$(MODDABLE)/build/bin/lin/release/
+NODE_MODULES=$(PWD)/node_modules
+ROOT=$(PWD)/../..
+
+$(NODE_MODULES)/.bin/xs-vat-worker: build/bin/lin/debug/xs-vat-worker
+	cp $< $@
+	echo "export const xsWorkerBin =\n  '$@';" >src/locate.js
+
+build/bin/lin/debug/xs-vat-worker: build $(TOOLS)/mcconfig moddable/xs/platforms/lin_xs_cli.c compartmap.json manifest.json
+	ROOT=$(ROOT) PATH=$(TOOLS):$$PATH MODDABLE=$(MODDABLE) mcconfig -o build -p x-cli-lin -m -d
+
+compartmap.json: src/vatWorker.js package.json
+	node -r esm tools/findmods.js $(ROOT) src/vatWorker.js >$@
+
+build:
+	mkdir -p build
+
+moddable/xs/platforms/lin_xs_cli.c: moddable/xs/platforms/lin_xs.h
+
+./moddable/README.md:
+	git submodule init
+	git submodule update
+
+moddable/xs/platforms/lin_xs.h: /usr/include/glib-2.0/gio/gio.h
+
+/usr/include/glib-2.0/gio/gio.h:
```

Comment:
let's put this into a separate `install-gio` target, with a note explaining to the user why their build is suddenly and mysteriously asking them for their root password. something like:

```make
.PHONY: install-gio
/usr/include/glib-2.0/gio/gio.h:
        @echo "GIO not installed, need root to run apt-get install libgio2.0-dev
        @echo "feel free to stop now and run `make install-gio` directly, then rebuild"
        $(MAKE) install-gio

install-gio:
        sudo apt-get -y update && sudo apt-get -y install libgio2.0-dev
```

Reply:
yes, that was sort of an executable note-to-self. Will do.